### DISCUSSION TITLE
Error message when using "getting started" instructions

### DISCUSSION THREAD

**Discussion by RogerPodacter** [shakacode/react_on_rails#1450] [Q&A] [answered]
Hi!

When I do the "getting started" steps from the documentation exactly on a new Rails 7 application I see this error: "undefined method `javascript_pack_tag' for #<ActionView::Base:0x0000000000b220>"

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/59190/165768798-b6c1eab7-249d-4861-8e02-5643a5dcd619.png">

Is this a known issue? I followed the instructions exactly on a new app. Thanks!

**Comment by jasonfb** [comment, marked as answer]
did you run `foreman start -f Procfile.dev-static` when you started the server?

**Reply by RogerPodacter** [reply]
No I think just `rails s`. Is that not supported?

**Reply by Judahmeek** [reply]
@RogerPodacter see #1485
